tests/test_views.py

```diff
@@ -4701,5 +4702,193 @@ def test_dashboard_institutions(self):
         assert_not_equal(dashboard_institutions[0]['id'], self.inst_four._id)
         assert_not_equal(dashboard_institutions[0]['id'], self.inst_five._id)
 
+
+class TestConfirmationViewBlockBingPreview(OsfTestCase):
```

Comment:
These tests (and the decorator `block_bing_preview`) will go away or be replaced when long-term fix for URL preview/prefetch is implemented in CAS-10 an OSF-7566.